### PR TITLE
Export data from spark to postgres

### PR DESCRIPTION
**Pull request by washort**
@robhudson this should give you a place to start, I wasn't able to get s3 access working with the older version of boto but boto3 seems to work fine.

### CODE REVIEW

File: notebooks/shield-dash-import.py [hunk 1]
```diff
@@ -22,8 +24,12 @@ def getShieldProps(p):
     }
     for k in ['firstrun', 'msg', 'name', 'variation']:
         if k == 'firstrun':
-            out[k] = datetime.datetime.utcfromtimestamp(
-                int(p['payload'][k]) // 1e3)
+            firstrun = p['payload'].get(k)
+            if firstrun is None:
+                out[k] = out['creation_date']
```
Comment (robhudson) [inline notebooks/shield-dash-import.py]: Was this change necessary? I'm not sure we want to conflate creation date with firstrun, which has a pretty specific use for these studies and isn't the same as creation_date.

Comment (washort) [inline notebooks/shield-dash-import.py]: I was getting KeyError on 'firstrun' for some of the data, it seems. So I hacked this temporarily. Not sure what the right solution is.

**Comment by robhudson** [comment]
@washort Updated and tested on spark cluster with success.

**Comment by washort** [comment]
r+
